src/mw/device/include/madrona/context.inl

```diff
@@ -82,6 +82,31 @@ inline void * Context::tmpAlloc(uint64_t num_bytes)
     return mwGPU::TmpAllocator::get().alloc(num_bytes);
 }
 
+template <typename... ComponentTs>
+inline Query<ComponentTs...> Context::query() {
+    return mwGPU::getStateManager()->query<ComponentTs...>();
+}
+    
+template <typename... ComponentTs, typename Fn>
+inline void Context::iterateQuery(Query<ComponentTs...> &query, Fn&& fn) {
+    mwGPU::getStateManager()->iterateQuery<sizeof...(ComponentTs)>(
+        world_id_.idx,
+        query.getSharedRef(),
+    [&](int32_t offset, auto ...raw_ptrs){
```

Comment:
I like this style, can you reduce tabbing of the lambda body below by one level though (just four spaces not 8)?